internal/commands/kubernetes/delete.go

```diff
@@ -0,0 +1,48 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/completion"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/resolver"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v4/upcloud/request"
+)
+
+// DeleteCommand creates the "kubernetes delete" command
+func DeleteCommand() commands.Command {
+	return &deleteCommand{
+		BaseCommand: commands.New(
+			"delete",
+			"Delete a Kubernetes cluster",
+			"upctl kubernetes delete 55199a44-4751-4e27-9394-7c7661910be3",
+			"upctl kubernetes delete my-kubernetes-cluster",
+		),
+	}
+}
+
+type deleteCommand struct {
+	*commands.BaseCommand
+	resolver.CachingKubernetes
+	completion.Kubernetes
+}
+
+// Execute implements commands.MultipleArgumentCommand
+func (s *deleteCommand) Execute(exec commands.Executor, arg string) (output.Output, error) {
+	svc := exec.All()
+	msg := fmt.Sprintf("Deleting Kubernetes cluster %v", arg)
+	exec.PushProgressStarted(msg)
+
+	err := svc.DeleteKubernetesCluster(&request.DeleteKubernetesClusterRequest{
+		UUID: arg,
+	})
+	if err != nil {
+		return commands.HandleError(exec, fmt.Sprintf("%s: failed", msg), err)
```

Comment:
Other `HandleError` calls do not add `:  failed` to the end of the status message anymore as failure is indicated with `✗` before the message.

Reply:
Fixed in https://github.com/UpCloudLtd/upcloud-cli/pull/190/commits/2e477343c90c637e44300d4a1d4cb81c7453fd03